Makefile

```diff
@@ -94,22 +94,30 @@ build-docs-capi:
 # We use cranelift as the default backend for the capi for now
 build-capi: build-capi-cranelift
```

Comment:
This command looks like it will fail now

Reply:
Thanks, fixed.